src/Sulu/Bundle/TranslateBundle/Command/ExportCommand.php

```diff
@@ -76,50 +64,93 @@ protected function configure()
                 'sets the filename of the exported file',
                 'sulu'
             );
+
+        $this->setDescription(
+            'Loads translation messages as defined by the parameters '
+            . 'from the bundles and writes them to a file, which also can be configured '
+            . 'via the parameters. If a locale got specified, only the translation '
+            . 'corresponding to that locale get exported. Otherwise the translation '
+            . 'files get exported for all locales defined in the system.'
+        );
     }
 
+    /**
+     * Loads translation messages as defined by the parameters from the bundles
+     * and writes them to a file, which also can be configured via the parameters.
+     * If a locale got specified, only the translation corresponding to that locale
+     * get exported. Otherwise the translation files get exported for all locales
+     * defined in the system.
+     *
+     * @param InputInterface $input The input of the command
+     * @param OutputInterface $output The output of the command
+     *
+     * @return int 0 iff everything went fine
+     */
     protected function execute(InputInterface $input, OutputInterface $output)
     {
         $locale = $input->getArgument('locale');
-        $format = $input->getArgument('format');
+        $locales = [];
+        if ($locale) {
+            array_push($locales, $locale);
+        } else {
+            $locales = array_merge($locales, $this->getContainer()->getParameter('sulu_core.translations'));
```

Comment:
No need for `array_merge`, since `$locales` will be empty anyway. Same for the `array_push` above. Maybe instantiate `$locales` as null first, and fill it accordingly in the if?